=== Docker Build Log for 6078248 ===
Started at: Mi 29. Okt 16:18:59 CET 2025

[2025-10-29 16:18:59] START: Building image for 6078248
#0 building with "default" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 266B done
#1 DONE 0.0s

#2 [internal] load metadata for docker.io/library/python:3.11
#2 DONE 0.0s

#3 [internal] load .dockerignore
#3 transferring context: 2B done
#3 DONE 0.0s

#4 [internal] load build context
#4 transferring context: 622B 0.0s done
#4 DONE 0.0s

#5 [1/6] FROM docker.io/library/python:3.11
#5 CACHED

#6 [2/6] COPY hard-gists/6078248/snippet.py /snippet.py
#6 DONE 0.1s

#7 [3/6] RUN ["pip","install","dj.choices"]
#7 1.905 Collecting dj.choices
#7 1.944   Downloading dj.choices-0.11.0.tar.gz (28 kB)
#7 1.962   Preparing metadata (setup.py): started
#7 2.348   Preparing metadata (setup.py): finished with status 'done'
#7 2.457 Collecting django (from dj.choices)
#7 2.466   Downloading django-5.2.7-py3-none-any.whl.metadata (4.1 kB)
#7 2.504 Collecting six (from dj.choices)
#7 2.512   Downloading six-1.17.0-py2.py3-none-any.whl.metadata (1.7 kB)
#7 2.556 Collecting asgiref>=3.8.1 (from django->dj.choices)
#7 2.563   Downloading asgiref-3.10.0-py3-none-any.whl.metadata (9.3 kB)
#7 2.600 Collecting sqlparse>=0.3.1 (from django->dj.choices)
#7 2.608   Downloading sqlparse-0.5.3-py3-none-any.whl.metadata (3.9 kB)
#7 2.633 Downloading django-5.2.7-py3-none-any.whl (8.3 MB)
#7 3.135    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 8.3/8.3 MB 16.6 MB/s eta 0:00:00
#7 3.143 Downloading six-1.17.0-py2.py3-none-any.whl (11 kB)
#7 3.159 Downloading asgiref-3.10.0-py3-none-any.whl (24 kB)
#7 3.175 Downloading sqlparse-0.5.3-py3-none-any.whl (44 kB)
#7 3.186    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 44.4/44.4 kB 4.0 MB/s eta 0:00:00
#7 3.223 Building wheels for collected packages: dj.choices
#7 3.224   Building wheel for dj.choices (setup.py): started
#7 3.405   Building wheel for dj.choices (setup.py): finished with status 'done'
#7 3.405   Created wheel for dj.choices: filename=dj_choices-0.11.0-py3-none-any.whl size=28716 sha256=8cc9cfc2d42a732ba509bd5db3a9120bfe920f7965263a4615fef89a7046ce14
#7 3.405   Stored in directory: /root/.cache/pip/wheels/97/38/bd/4ca089ff8ffdead6a07e7af033d36d74a01ab7a37e70797942
#7 3.407 Successfully built dj.choices
#7 3.455 Installing collected packages: sqlparse, six, asgiref, django, dj.choices
#7 4.942 Successfully installed asgiref-3.10.0 dj.choices-0.11.0 django-5.2.7 six-1.17.0 sqlparse-0.5.3
#7 4.942 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#7 5.038 
#7 5.038 [notice] A new release of pip is available: 24.0 -> 25.3
#7 5.038 [notice] To update, run: pip install --upgrade pip
#7 DONE 5.4s

#8 [4/6] RUN ["pip","install","indras_net"]
#8 0.745 Collecting indras_net
#8 0.902   Downloading indras_net-2.0.6.tar.gz (48 kB)
#8 0.930      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 48.6/48.6 kB 1.5 MB/s eta 0:00:00
#8 0.940   Preparing metadata (setup.py): started
#8 1.058   Preparing metadata (setup.py): finished with status 'done'
#8 1.108 Collecting networkx (from indras_net)
#8 1.117   Downloading networkx-3.5-py3-none-any.whl.metadata (6.3 kB)
#8 1.345 Collecting numpy (from indras_net)
#8 1.355   Downloading numpy-2.3.4-cp311-cp311-manylinux_2_27_x86_64.manylinux_2_28_x86_64.whl.metadata (62 kB)
#8 1.379      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.1/62.1 kB 2.4 MB/s eta 0:00:00
#8 1.417 Downloading networkx-3.5-py3-none-any.whl (2.0 MB)
#8 1.500    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.0/2.0 MB 24.9 MB/s eta 0:00:00
#8 1.510 Downloading numpy-2.3.4-cp311-cp311-manylinux_2_27_x86_64.manylinux_2_28_x86_64.whl (16.9 MB)
#8 2.203    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 16.9/16.9 MB 21.2 MB/s eta 0:00:00
#8 2.255 Building wheels for collected packages: indras_net
#8 2.260   Building wheel for indras_net (setup.py): started
#8 2.439   Building wheel for indras_net (setup.py): finished with status 'done'
#8 2.439   Created wheel for indras_net: filename=indras_net-2.0.6-py3-none-any.whl size=76537 sha256=857e980dddcf67133995bf9111ca0e391b107e56dd29a857564ab70fad6bd4d8
#8 2.439   Stored in directory: /root/.cache/pip/wheels/5c/22/97/5567dece42982388902a2ab5839e2ff0775ee3608834b5e431
#8 2.441 Successfully built indras_net
#8 2.514 Installing collected packages: numpy, networkx, indras_net
#8 4.874 Successfully installed indras_net-2.0.6 networkx-3.5 numpy-2.3.4
#8 4.874 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#8 4.879 
#8 4.879 [notice] A new release of pip is available: 24.0 -> 25.3
#8 4.879 [notice] To update, run: pip install --upgrade pip
#8 DONE 5.5s

#9 [5/6] RUN ["pip","install","Django"]
#9 0.447 Requirement already satisfied: Django in /usr/local/lib/python3.11/site-packages (5.2.7)
#9 0.450 Requirement already satisfied: asgiref>=3.8.1 in /usr/local/lib/python3.11/site-packages (from Django) (3.10.0)
#9 0.451 Requirement already satisfied: sqlparse>=0.3.1 in /usr/local/lib/python3.11/site-packages (from Django) (0.5.3)
#9 0.516 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#9 0.521 
#9 0.521 [notice] A new release of pip is available: 24.0 -> 25.3
#9 0.521 [notice] To update, run: pip install --upgrade pip
#9 DONE 0.6s

#10 [6/6] RUN ["pip","install","models"]
#10 0.646 Collecting models
#10 0.684   Downloading models-0.9.3.tar.gz (16 kB)
#10 0.700   Preparing metadata (setup.py): started
#10 0.794   Preparing metadata (setup.py): finished with status 'error'
#10 0.797   error: subprocess-exited-with-error
#10 0.797   
#10 0.797   × python setup.py egg_info did not run successfully.
#10 0.797   │ exit code: 1
#10 0.797   ╰─> [8 lines of output]
#10 0.797       Traceback (most recent call last):
#10 0.797         File "<string>", line 2, in <module>
#10 0.797         File "<pip-setuptools-caller>", line 34, in <module>
#10 0.797         File "/tmp/pip-install-9h1q20rj/models_8a9c7160563a46478ed999f43054dfe4/setup.py", line 25, in <module>
#10 0.797           import models
#10 0.797         File "/tmp/pip-install-9h1q20rj/models_8a9c7160563a46478ed999f43054dfe4/models/__init__.py", line 23, in <module>
#10 0.797           from base import *
#10 0.797       ModuleNotFoundError: No module named 'base'
#10 0.797       [end of output]
#10 0.797   
#10 0.797   note: This error originates from a subprocess, and is likely not a problem with pip.
#10 0.799 error: metadata-generation-failed
#10 0.799 
#10 0.799 × Encountered error while generating package metadata.
#10 0.799 ╰─> See above for output.
#10 0.799 
#10 0.799 note: This is an issue with the package mentioned above, not pip.
#10 0.799 hint: See above for details.
#10 0.803 
#10 0.803 [notice] A new release of pip is available: 24.0 -> 25.3
#10 0.803 [notice] To update, run: pip install --upgrade pip
#10 ERROR: process "pip install models" did not complete successfully: exit code: 1
------
 > [6/6] RUN ["pip","install","models"]:
0.799 error: metadata-generation-failed
0.799 
0.799 × Encountered error while generating package metadata.
0.799 ╰─> See above for output.
0.799 
0.799 note: This is an issue with the package mentioned above, not pip.
0.799 hint: See above for details.
0.803 
0.803 [notice] A new release of pip is available: 24.0 -> 25.3
0.803 [notice] To update, run: pip install --upgrade pip
------
Dockerfile:6
--------------------
   4 |     RUN ["pip","install","indras_net"]
   5 |     RUN ["pip","install","Django"]
   6 | >>> RUN ["pip","install","models"]
   7 |     CMD ["python3","/snippet.py"]
   8 |     
--------------------
ERROR: failed to build: failed to solve: process "pip install models" did not complete successfully: exit code: 1

View build details: docker-desktop://dashboard/build/default/default/1w4ik0xz98rkumeqx3ocg3pnx
[2025-10-29 16:19:12] ERROR: Build failed for 6078248
